src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -589,6 +600,7 @@ private async Task<List<string>> GetRequiredResourceProvidersNotRegisteredAsync(
 
         private async Task ConfigureVmAsync(ConnectionInfo sshConnectionInfo)
         {
+            await ExecuteCommandOnVirtualMachineAsync(sshConnectionInfo, $"echo '{configuration.VmPassword}' | sudo -S -p '' /bin/bash -c \"echo '{configuration.VmUsername} ALL=(ALL) NOPASSWD:ALL' > /etc/sudoers.d/z_{configuration.VmUsername}\"");
```

Comment:
The system determines if the user has ability to act as root by means of the /etc/sudoers file and /etc/sudoers.d directory. If the user is listed there, the user can run as root and the entries in the files determine what the user can do. sudoers file is processed first and it includes a "link" to sudoers.d directory. The files in the directory are then processed in alphabetical order. Entries for the same user in those in preceding files. NOPASSWD:ALL setting indicates that the user will not be prompted for password when running sudo command. This is the default setting for the initial user (default vmadmin) when the VM is first created. If user interacts with the portal to change the password or create new user, a file named "waagent" is added to sudoers.d directory with a line "<username> ALL = (ALL) ALL" for each user. In case of a password change for the initial user, this overrides the existing setting for the initial user in file "90-cloud-init-users" containing "vmadmin ALL=(ALL) NOPASSWD:ALL". In case of the new user, the only entry is the line in waagent file. The fix merely adds a file named "z_<username>" with the content "<username> ALL=(ALL) NOPASSWD:ALL", for the user running the deployer, and overrides any setting in preceding files. After this, any subsequent sudo commands will not prompt for password. -S forces the sudo to read the password from stdin, -p sets the sudo password prompt to an empty string.